bayes/latent.py

```diff
@@ -174,6 +174,13 @@ def update(self, number_vector):
         for latent in self.values():
             latent.update(number_vector)
 
+    @property
+    def N(self):
```

Comment:
I would suggest using a more intuitive name (dim_parameter_vector?) or len_parameter_vector.

Reply:
There is the `__len__` method that allows us to simply write
~~~py
print(len(problem.latent))
~~~

?

Reply:
but that returns the number of variables not N (different, when at least one variable is a vector).